assets/src/edit-story/components/keyboard/keys.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @todo: additional format for metaOrCntrlKey for MacOS/other or make it
+// automatic.
+const keys = {
```

Comment:
I thought the idea here was to allow component to register their keyboard shortcuts?

Reply:
That's not ruled out though by this PR, is it?
Haven't tested it yet, but as a prototype this PR seems like a great first step.

Reply:
Yeah. Sorry, this is to address any kind of localization or customization. Though, bunch of localization can be done now at the call site too. I think we'll figure out as we go where this is most convenient.